Additional help text for commands

pkess opened the pull request:
Hey,

by using beets on my computer I experienced the lack of some sample queries when doing beet ls. Especially the format option was difficult to me.
Maybe I can help others with this code.

sampsyo commented on beets/ui/commands.py:
```diff
@@ -973,6 +973,8 @@ def list_func(lib, opts, args):
 
 list_cmd = ui.Subcommand('list', help='query the library', aliases=('ls',))
+list_cmd.parser.add_help_text('query the library')
+list_cmd.parser.add_help_text('Example usage: %prog -f \'$album: $title\' artist:foo')
```
Maybe just directly modifying `list_cmd.parser.usage` would be simpler?

sampsyo replied:
Also, it would be awesome to follow the "no contrived examples" rule and use a real artist name (or fragment of an artist name) instead of "foo". So, perhaps, `artist:beatles` or something.

sampsyo commented:
Seems like a good idea! I left a couple of comments, and it seems that Travis found some style suggestions too.

pkess commented:
now I changed the code and travis seems to be ok.
I also changed the artist to suggested beatles.

I did not get your point with list_cmd.parser.usage

sampsyo commented:
Thanks! I just meant to avoid creating a new method since accessing the member directly seems clean enough. See the above commit.
